app/config/variables.php

```diff
@@ -306,24 +306,24 @@
                 'question' => '',
                 'filter' => 'password'
             ],
-            [
-                'name' => '_APP_DB_PROJECT',
-                'description' => 'A list of comma-separated key value pairs representing Project DBs where key is the database name and value is the DSN connection string.',
-                'introduction' => 'TBD',
-                'default' => 'db_fra1_01=mysql://user:password@mariadb:3306/appwrite',
-                'required' => true,
-                'question' => '',
-                'filter' => ''
-            ],
-            [
-                'name' => '_APP_DB__APP_DB_CONSOLEROOT_PASS',
-                'description' => 'A key value pair representing the Console DB where key is the database name and value is the DSN connection string.',
-                'introduction' => 'TBD',
-                'default' => 'db_fra1_01=mysql://user:password@mariadb:3306/appwrite',
-                'required' => true,
-                'question' => '',
-                'filter' => ''
-            ]
+            // [
+            //     'name' => '_APP_CONNECTIONS_DB_PROJECT',
+            //     'description' => 'A list of comma-separated key value pairs representing Project DBs where key is the database name and value is the DSN connection string.',
```

Comment:
We need to include the new env vars in here.

Reply:
I'm hiding them all for now as we don't need them in the docs.

Reply:
okay 👍